### PR TITLE
CLI 3.10 - Release

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Added new feature 'Tags' in CLI

### CODE REVIEW

File: advocacy_docs/edb-postgres-ai/cloud-service/using_cluster/cli/managing_clusters.mdx [hunk 1]
```diff
@@ -87,24 +87,29 @@ Here's a sample configuration file in YAML format with Azure specified as the pr
...
+provider: azure                                                       # <string:  cloud provider id, valid values - Deployment: Your Cloud Account ["azure", "aws", "gcp"], Deployment: BigAnimal ["bah:azure", "bah:aws", "bah:gcp"]>
+cspSubscriptionIDs:                                                   # <list:  cloud provider subscription/account ID, required when cluster is BigAnimal hosted>
   - 123123123                                                         # <string>
   - 456456456                                                         # <string>
-serviceAccountIds:                                                    # <list: A Google Cloud Service Account is used for logs. If you leave this blank, then you will be unable to access log details for this cluster. Required when cluster is EDB Hosted>
+serviceAccountIds:                                                    # <list: A Google Cloud Service Account is used for logs. If you leave this blank, then you will be unable to access log details for this cluster. Required when cluster is BigAnimal hosted>
```
Comment (gvasquezvargas) [inline advocacy_docs/edb-postgres-ai/cloud-service/using_cluster/cli/managing_clusters.mdx:94]: Is there a reason why we are calling it BA hosted here?

File: advocacy_docs/edb-postgres-ai/cloud-service/using_cluster/cli/managing_clusters.mdx [hunk 2]
```diff
@@ -87,24 +87,29 @@ Here's a sample configuration file in YAML format with Azure specified as the pr
...
 clusterName: biganimal_cluster                                        # <string:  cluster name>
 password: ************                                                # <string:  cluster password (must be at least 12 characters)>
-# refer following link for steps to setup IAM:  https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/01_postgres_access/iam-authentication-for-postgres
+tags:                                                                 # <list: Group clusters and organise across different resource types. This will help to manage resources effectively.>
+  - name: tag1                                                        # <string: assign "tag1" property to the cluster>
+    color: "#c5eae7"                                                  # <string: color for "tag1">
+  - name: tag2                                                        # <string: assign "tag2" property to the cluster>
+    color: "#0b6ff4"                                                  # <string: color for "tag2">
+# refer following link for steps to setup IAM:  https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/01_postgres_access/iam-authentication-for-postgres
```
Comment (gvasquezvargas) [inline advocacy_docs/edb-postgres-ai/cloud-service/using_cluster/cli/managing_clusters.mdx:104]: Wrong link, maybe it should be https://www.enterprisedb.com/docs/edb-postgres-ai/cloud-service/using_cluster/postgres_access/database_authentication/#iam-authentication-for-postgres ?

File: advocacy_docs/edb-postgres-ai/cloud-service/using_cluster/cli/managing_clusters.mdx [hunk 3]
```diff
@@ -117,9 +122,21 @@ pgConfigMap:                                                          # <Object:
...
 backupRetentionPeriod: 30d                                            # <string: Retention period must be between 1-180 days or 1-25 weeks or 1-6 months. Using strings like "7d" or "2w" or "3m" to specify days, weeks and months respectively.>
-superuserAccess: true                                                 # <bool:    Superuser Access, Grants superuser access to the edb_admin role, this will bypass all the permission checks and should be used very carefully.>
-pgvector: true                                                      # <bool: Set True to enable pgvector extension. Adds support for vector storage and vector similarity search to Postgres.>
-postgis: true                                                       # <bool: Set True to enable postgis extension. PostGIS extends the capabilities of the PostgreSQL relational database by adding support storing, indexing and querying geographic data.>
+customMaintenanceWindow:                                              # <Object:  Schedule a custom maintenance window>
+  isEnabled: true                                                     # <bool: Set True to enable custom maintenance window>
```
Comment (gvasquezvargas) [inline advocacy_docs/edb-postgres-ai/cloud-service/using_cluster/cli/managing_clusters.mdx:126]:
```suggestion
  isEnabled: true                                                     # <bool: Set to True to enable custom maintenance window>
```

File: advocacy_docs/edb-postgres-ai/cloud-service/using_cluster/cli/managing_clusters.mdx [hunk 4]
```diff
@@ -117,9 +122,21 @@ pgConfigMap:                                                          # <Object:
...
+  maintenanceStartDay: Monday                                         # <string:  Select desired day (e.g. monday) from a week for maintenance activities>
```
Comment (gvasquezvargas) [inline advocacy_docs/edb-postgres-ai/cloud-service/using_cluster/cli/managing_clusters.mdx:127]:
```suggestion
  maintenanceStartDay: Monday                                         # <string:  Select desired day (e.g. Monday) in a week for maintenance activities>
```

File: advocacy_docs/edb-postgres-ai/cloud-service/using_cluster/cli/managing_clusters.mdx [hunk 5]
```diff
@@ -87,24 +87,29 @@ Here's a sample configuration file in YAML format with Azure specified as the pr
 ---
 clusterArchitecture: ha                                               # <string:  cluster architecture, valid values ["single" | "ha" ]>
 haStandbyReplicas: 2                                                  # <number:  Number of standby replicas. Field must be specified if user has selected Primary/Standby High Availability cluster type. Default value is 2, valid values [1, 2].>
-provider: azure                                                       # <string:  cloud provider id, valid values ["azure", "aws", "gcp"]>
-cspSubscriptionIDs:                                                   # <list:  cloud provider subscription/account ID, required when cluster is EDB Hosted>
+provider: azure                                                       # <string:  cloud provider id, valid values - Deployment: Your Cloud Account ["azure", "aws", "gcp"], Deployment: BigAnimal ["bah:azure", "bah:aws", "bah:gcp"]>
```
Comment (gvasquezvargas) [inline advocacy_docs/edb-postgres-ai/cloud-service/using_cluster/cli/managing_clusters.mdx:90]: Are we going back to calling it BA instead of CS?

**Review by gvasquezvargas — COMMENTED** [review]
Adding unfished review to see comments when implementing locally. Commit with fixes coming

**Review by edbamolvyavahare — COMMENTED** [review]
can take care of BigAnimal hosted -> EDB hosted here
note: Currently, rebranding BigAnimal CLI is yet to be done, but if we dont want BA hosted in official docs lets remove it.

**Comment by gvasquezvargas** [comment]
@edbamolvyavahare removed mentions of BigAnimal whenever not in commands, also updated a broken link to point to the correct page in the docs online 104, which I think we'll want to correct some time on the code as well. If it looks good to you now, could you please approve? TY!

**Comment by edbamolvyavahare** [comment]
> @edbamolvyavahare removed mentions of BigAnimal whenever not in commands, also updated a broken link to point to the correct page in the docs on line 104, which I think we'll want to correct some time on the code as well. If it looks good to you now, could you please approve it? TY!

Sure taking a look, Thanks.

**Review by edbamolvyavahare — APPROVED** [review]
LGTM, please check with me before publishing these changes; CLI v3.10 release is planned today (yet to be done)

**Review by gvasquezvargas — APPROVED** [review]
LGTM, awaiting green light from team to publish.

**Comment by edbamolvyavahare** [comment]
@nidhibhammar @gvasquezvargas 
CLI v3.10.0 has been released; you may please publish these now.

**Pull request merged**